python/cudf_polars/tests/experimental/test_parallel.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import polars as pl
+from polars import GPUEngine
+from polars.testing import assert_frame_equal
+
+from cudf_polars import Translator
+from cudf_polars.experimental.parallel import evaluate_dask
+from cudf_polars.testing.asserts import Executor
+
+
+def test_evaluate_dask():
+    df = pl.LazyFrame({"a": [1, 2, 3], "b": [3, 4, 5], "c": [5, 6, 7], "d": [7, 9, 8]})
+    q = df.select(pl.col("a") - (pl.col("b") + pl.col("c") * 2), pl.col("d")).sort("d")
+
+    config = GPUEngine(raise_on_fail=True, executor=Executor)
+    qir = Translator(q._ldf.visit(), config).translate_ir()
+
+    expected = qir.evaluate(cache={}).to_polars()
+    got = evaluate_dask(qir).to_polars()
```

Comment:
Can we not test these like so:

```
expected = q.collect(engine="cpu")
got_gpu = q.collect(engine=pl.GPUEngine(raise_on_fail=True))
got_dask = q.collect(engine=pl.GPUEngine(raise_on_fail=True, executor="dask-experimental")
assert_frame_equal(expected, got_gpu)
assert_frame_equal(expected, got_dask)
```
?

Reply:
Done in https://github.com/rapidsai/cudf/pull/17262/commits/7febe212a7fdd7435c2e4ddf03229036c7ce7782 .